pkg/blobinfocache/memory/memory.go

```diff
@@ -18,13 +18,18 @@ type locationKey struct {
 	blobDigest digest.Digest
 }
 
+type knownLocationInfo struct {
+	when           time.Time
+	compressorName string
+}
+
 // cache implements an in-memory-only BlobInfoCache
 type cache struct {
 	mutex sync.Mutex
 	// The following fields can only be accessed with mutex held.
 	uncompressedDigests   map[digest.Digest]digest.Digest
-	digestsByUncompressed map[digest.Digest]map[digest.Digest]struct{}             // stores a set of digests for each uncompressed digest
-	knownLocations        map[locationKey]map[types.BICLocationReference]time.Time // stores last known existence time for each location reference
+	digestsByUncompressed map[digest.Digest]map[digest.Digest]struct{}                     // stores a set of digests and media types for each uncompressed digest
```

Comment:
(The comment update is incorrect.)

Reply:
Whoops, leftover from a previous iteration.  Reverting this bit.